Classes/EventStore/EventFromStream.php

```diff
@@ -0,0 +1,86 @@
+<?php
+namespace Neos\Cqrs\EventStore;
+
+/*
+ * This file is part of the Neos.EventStore package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+final class EventFromStream
```

Comment:
Here also we need to take a decision about how to name object -

- describe what the object IS or DO

EventFromStream vs EventStream ... confusing